molecule/release-static/molecule.yml

```diff
@@ -62,4 +62,5 @@ scenario:
     - side_effect
     - verify
 verifier:
-  name: inspec
+  name: ansible
+  directory: tests
```

Comment:
Wow, you managed to fix the inspec tests not running on Python 2.

You need to update this:
https://pulp-installer.readthedocs.io/en/latest/contributing/#molecule-limitations

And I wonder if we even need to installer molecule-inspec. Does it only help with generating the files we need? If so, we should reference that in our contributing.md as a reference.

Is there any reduced functionality when running the tests though?

Reply:
Was this a python2 problem? Interesting. But now that you say that, the vm i used to run molecule is using python2. And i must say, i have no idea, how this was supposed to work with `name: inspec`.
Yes, the verify playbook invests quite some time in installing the inspec on it's own.

Reply:
I split the "Do not install molecule-inspec" into a separate commit for easier review.

Reply:
@mdellweg I was about to answer your question about how it works, but then I noticed I do not see any commit or PR about this.

Reply:
Oh right. That one was adopted by @fao89 in his fixing the CI Marathon.